packages/web/src/common/services/audius-backend/AudiusBackend.ts

```diff
@@ -1142,12 +1142,17 @@ export const audiusBackend = ({
     metadata: TrackMetadata,
     onProgress: (loaded: number, total: number) => void
   ) {
-    return await audiusLibs.Track.uploadTrack(
+    const isTrackEntityManagerEnabled =
+      getFeatureEnabled(FeatureFlags.TRACK_ENTITY_MANAGER_ENABLED) ?? false
+
+    // TODO: Implement libs track entity manager
+    return await audiusLibs.Track.uploadTrack({
       trackFile,
       coverArtFile,
       metadata,
+      useEntityManager: isTrackEntityManagerEnabled,
       onProgress
-    )
+    })
```

Comment:
do we also need uploadTrackToCreatorNode and deleteTrack below?

https://github.dev/AudiusProject/audius-client/blob/jowlee-track-entity-manager/packages/web/src/common/services/audius-backend/AudiusBackend.ts#L1166-L1167

https://github.dev/AudiusProject/audius-client/blob/jowlee-track-entity-manager/packages/web/src/common/services/audius-backend/AudiusBackend.ts#L1798-L1799

Reply:
`uploadTrackContentToCreatorNode` strictly uploads it to the content node, it doesn't have contract interactions
But yes, good callout on the delete track